README.md.in

```diff
@@ -0,0 +1,92 @@
+[![Build Status](https://jenkins.ovirt.org/job/oVirt_ovirt-ansible-collection_standard-check-pr/badge/icon)](https://jenkins.ovirt.org/job/oVirt_ovirt-ansible-collection_standard-check-pr/)
+[![Build Status](https://img.shields.io/badge/docs-latest-blue.svg)](https://docs.ansible.com/ansible/2.10/collections/ovirt/ovirt/index.html)
+
+oVirt ansible collection
+====================================
+
+The `@NAMESPACE@.@NAME@` manages all ansible modules of oVirt.
+
+The pypi installation is no longer supported if you want
+to install all dependencies do it manually or install the
+collection from RPM and it will be done automatically.
+
+Note
+----
+Please note that when installing this collection from Ansible Galaxy you are instructed to run following command:
+
+```bash
+$ ansible-galaxy collection install @NAMESPACE@.@NAME@
```

Comment:
Is this command the same for Automation Hub?

Reply:
yes, from AH you install it by `ansible-galaxy collection install redhat.rhv`